src/pyedb/dotnet/edb.py

```diff
@@ -2232,7 +2232,8 @@ def _create_cutout_multithread(
             )
             if extent_type in ["Conforming", self.edb_api.geometry.extent_type.Conforming, 1] and extent_defeature > 0:
                 _poly = _poly.Defeature(extent_defeature)
-
+            if extent_type in ["Conforming", self.edb_api.geometry.extent_type.Conforming, 1]:
+                _poly = _poly.CreateFromArcs(_poly.GetArcData(), True)
```

Comment:
Can we leverage the previous if statement to avoid "duplicate" ? This would change things into something like:
```python
if extent_type in ["Conforming", self.edb_api.geometry.extent_type.Conforming, 1]:
    if extent_defeature > 0:
        _poly = _poly.Defeature(extent_defeature)
    _poly = _poly.CreateFromArcs(_poly.GetArcData(), True)
```

Reply:
agree